From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Tue, 11 Oct 2011 09:27:48 -0600
Patch-mainline: v3.2-rc1
Subject: [PATCH] Staging: hv: remove vmbus_loglevel as it is not used at all anymore
Git-commit: 815166b95df1acb4890e9dbdb26660e9c00a7505

As there is no user of this variable, it's time to delete it.  For
dynamic debugging of the hyperv code, use the standard dynamic debug
kernel interface.

Cc: K. Y. Srinivasan <kys@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/hv/vmbus_drv.c |    5 -----
 include/linux/hyperv.h |    5 -----
 2 files changed, 0 insertions(+), 10 deletions(-)

diff --git a/drivers/hv/vmbus_drv.c b/drivers/hv/vmbus_drv.c
index bb9f205..e648571 100644
--- a/drivers/hv/vmbus_drv.c
+++ b/drivers/hv/vmbus_drv.c
@@ -41,10 +41,6 @@ static struct acpi_device  *hv_acpi_dev;
 
 static struct tasklet_struct msg_dpc;
 static struct tasklet_struct event_dpc;
-
-unsigned int vmbus_loglevel = (ALL_MODULES << 16 | INFO_LVL);
-EXPORT_SYMBOL(vmbus_loglevel);
-
 static struct completion probe_event;
 static int irq;
 
@@ -767,6 +763,5 @@ cleanup:
 
 MODULE_LICENSE("GPL");
 MODULE_VERSION(HV_DRV_VERSION);
-module_param(vmbus_loglevel, int, S_IRUGO|S_IWUSR);
 
 module_init(hv_acpi_init);
diff --git a/include/linux/hyperv.h b/include/linux/hyperv.h
index 4fa32ec..7211e2c 100644
--- a/include/linux/hyperv.h
+++ b/include/linux/hyperv.h
@@ -720,11 +720,6 @@ extern void vmbus_ontimer(unsigned long data);
 							INPUTVSC_DRV|\
 							BLKVSC_DRV)
 
-/* Logging Level */
-#define INFO_LVL				6
-
-extern unsigned int vmbus_loglevel;
-
 struct hv_driver;
 struct hv_device;
 
-- 
1.6.0.2


